### PR TITLE
Merge 6 -> 7

### PR DESCRIPTION
**Pull request by scpeters**
# ➡️ Forward port

Port `ign-gui6` to `main`

Branch comparison: https://github.com/ignitionrobotics/ign-gui/compare/main...ign-gui6

**Note to maintainers**: Remember to **Merge** with commit (not squash-merge or rebase)

### CODE REVIEW

**Comment by scpeters** [comment]
needed by https://github.com/ignitionrobotics/ign-gazebo/pull/1183

File: README.md [hunk 1]
```diff
@@ -9,10 +9,10 @@
...
-Homebrew      | [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=ignition_gui-ci-main-homebrew-amd64)](https://build.osrfoundation.org/job/ignition_gui-ci-main-homebrew-amd64)
-Windows       | [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=ign_gui-ci-win)](https://build.osrfoundation.org/job/ign_gui-ci-win)
+Test coverage | [![codecov](https://codecov.io/gh/ignitionrobotics/ign-gui/branch/ign-gui6/graph/badge.svg)](https://codecov.io/gh/ignitionrobotics/ign-gui/branch/ign-gui6)
+Ubuntu Bionic | [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=ignition_gui-ci-ign-gui6-bionic-amd64)](https://build.osrfoundation.org/job/ignition_gui-ci-ign-gui6-bionic-amd64)
+Homebrew      | [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=ignition_gui-ci-ign-gui6-homebrew-amd64)](https://build.osrfoundation.org/job/ignition_gui-ci-ign-gui6-homebrew-amd64)
+Windows       | [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=ign_gui-ign-6-win)](https://build.osrfoundation.org/job/ign_gui-ign-6-win)
```
Comment (jennuine) [inline README.md:15]: I think these changes need to be reverted

**Review by jennuine — APPROVED** [review]
LGTM, one minor comment

File: README.md (hunk 1 above)
Comment (scpeters) [reply README.md:15]: agreed

Comment (scpeters) [reply README.md]: fixed in a force-push amend: https://github.com/ignitionrobotics/ign-gui/pull/316/commits/41e1e529422ee439c201ac2b5a240e9bac533881

**Comment by scpeters** [comment]
some github actions test failures, but otherwise clean; I will merge now

**Pull request merged**